pds_doi_core/input/osti_input_validator.py

```diff
@@ -67,27 +74,15 @@ def validate(self, input_to_osti):
             logger.error(msg)
             raise InputFormatException(msg)
 
-        # Check 2. Bad tag(s) in <record> element, e.g. status='Release'.  It should only be status='Pending'
-        # Check 3. Bad date formats.
-        date_fields_to_check = ['publication_date', 'date_record_added', 'date_record_updated']
+        # Check 2. Bad tag(s) in <record> element, e.g. status='Release'.  It should only be in possible_status_list variable.
```

Comment:
I am OK with doing some validation with our own code because it is our code that is adding these different status values.  Modifying the XSD present its own challenge as we are not the custodian.  If we get another version from soneone else, it would be hard to keep track of the changes we've made in the current version.

With regard to 'Pending' status, I think this is what OSTI send back when you send them something.  Sometimes if their system is happy, they may send back immediately 'Registered'.  For 'Reserved' this is what we send to them in the record for 'reserve' action.  They may send back 'Pending' because it takes some time for the record to make its way through their system.

Reply:
We can keep the validation in the code since we can not manage case in xsd, but we need to think of the xsd and schematron as ours. Ron is part of the team and if he is asked to update the version he will start from our version.